test/uk/gov/hmrc/pegaproofofconceptproxy/controllers/PegaProxyControllerSpec.scala

```diff
@@ -103,12 +145,25 @@ class PegaProxyControllerSpec extends AnyWordSpec with Matchers with GuiceOneApp
 
         verify(
           getRequestedFor(urlPathEqualTo(url))
-            .withHeader(HeaderNames.AUTHORIZATION, equalTo("Basic dToxMjM0"))
+            .withHeader(HeaderNames.AUTHORIZATION, equalTo("Bearer dToxMjM0"))
         )
       }
 
       "return an error status if something wrong with response status not 200" in {
 
+        stubFor(
```

Comment:
it could be worth doing an small additional test to see what happens when the call to get a token fails just so we've got maximum coverage

Reply:
okay will do